src/main/java/Duke.java

```diff
@@ -1,12 +1,18 @@
 import java.io.*;
```

Comment:
Good usage of individual imports! For this particular import for io, could this be extracted out to be individual imports as well ?